lambdas/check-mobile-app-vc-receipt/src/test/java/uk/gov/di/ipv/core/checkmobileappvcreceipt/CheckMobileAppVcReceiptHandlerTest.java

```diff
@@ -81,37 +109,61 @@ void shouldReturn500WhenCriResponseNotFound() throws Exception {
 
         // Act
         var response = checkMobileAppVcReceiptHandler.handleRequest(requestEvent, mockContext);
+        var journeyResponse =
+                OBJECT_MAPPER.readValue(response.getBody(), JourneyErrorResponse.class);
 
         // Assert
         assertEquals(HttpStatus.SC_INTERNAL_SERVER_ERROR, response.getStatusCode());
+        assertEquals(
+                new JourneyErrorResponse(
+                        JOURNEY_ERROR_PATH,
+                        HttpStatus.SC_INTERNAL_SERVER_ERROR,
+                        ErrorResponse.CRI_RESPONSE_ITEM_NOT_FOUND),
+                journeyResponse);
     }
 
     @Test
     void shouldReturn200WhenStatusNotPending() throws Exception {
         // Arrange
         var requestEvent = buildValidRequestEventWithState();
-        when(ipvSessionService.getIpvSession(TEST_IPV_SESSION_ID))
-                .thenReturn(buildValidIpvSessionItem());
+        var ipvSessionItem = buildValidIpvSessionItem();
+        var clientOAuthSessionItem = buildValidClientOAuthSessionItem();
+        when(ipvSessionService.getIpvSession(TEST_IPV_SESSION_ID)).thenReturn(ipvSessionItem);
         when(clientOAuthSessionDetailsService.getClientOAuthSession(TEST_CLIENT_OAUTH_SESSION_ID))
                 .thenReturn(buildValidClientOAuthSessionItem());
         when(criResponseService.getCriResponseItem(TEST_USER_ID, Cri.DCMAW_ASYNC))
                 .thenReturn(buildValidCriResponseItem(CriResponseService.STATUS_RECEIVED));
```

Comment:
Hmm I don't think STATUS_RECEIVED is ever actually used? we just keep the CRI record in status pending even once we've received the VC (rightly or wrongly 🤔)